apps/pie-storybook/stories/pie-card-container.stories.ts

```diff
@@ -59,6 +60,26 @@ const cardContainerStoryMeta: CardContainerStoryMeta = {
                 summary: 'default',
             },
         },
+        padding: {
+            description: 'Set the padding of the card.',
+            control: {
+                type: 'text',
+            },
+            options: padding,
+            defaultValue: {
+                summary: 'none',
```

Comment:
none isn't a valid value for your type definition, maybe we can default it to '' or remove the defaultValue  here at all since the prop is optional